dkist/dataset/tiled_dataset.py

```diff
@@ -152,11 +152,25 @@ def tiles_shape(self):
         """
         return [[tile.data.shape for tile in row] for row in self]
 
-    def plot(self, slice_index: int, share_zscale=False, **kwargs):
+    def plot(self, slice_index, share_zscale=False, **kwargs):
+        """
+        Plot a slice of each tile in the TiledDataset
+
+        Parameters
+        ----------
+        slice_index : `int`, sequence of `int`s or `numpy.s_`
```

Comment:
Is this the same as the input to `slice_tiles`? if yes it should be referenced from here. Also that needs a docstring now I look at it.